### PR TITLE
Fix typo in test_sp/test_mpitests.py

### PR DESCRIPTION
**Pull request by niltonlk**
Fix issue #1879 
A typo in line 41 of test_sp/test_mpitests.py that reads "python" instead of "nosetests" causes the test to fail.
Note that the error message that should be displayed in case of failure in line 50 reads "nosetests".
Test pass by fixing this typo.

### CODE REVIEW

**Comment by stinebuu** [comment]
@jougs Could you take a look at this, it seems fine to me, but I was suddenly afraid it might break something.

**Review by jougs — APPROVED** [review]
LGTM. Thanks!

Merging without formal second review, as @stinebuu indicated approval and the change is pretty small.

**Pull request merged**
